### PR TITLE
Edits to FPGA readme files for format and text flow.

### PR DESCRIPTION
**Pull request by MCamp859**
Signed-off-by: MCamp859 <mary.camp@ptiglobal.net>

### CODE REVIEW

File: cmd/fpga_plugin/README.md [hunk 1]
```diff
@@ -1,27 +1,40 @@
...
 ```
 $ ls /var/lib/kubelet/device-plugins/kubelet.sock
 /var/lib/kubelet/device-plugins/kubelet.sock
 ```
 
+### Choose mode for FPGA device plugin
+
+You can run the FPGA device plugin in either afu or region mode.
+
 #### Run FPGA device plugin in afu mode
```
[the remark this answers is not in the document]
Comment (bart0sh) [reply cmd/fpga_plugin/README.md]: I think 'af' should be here. Here is a quote from plugin help output:
```
./fpga_plugin --help
...
  -mode string
        device plugin mode: 'af' (default), 'region' or 'regiondevel' (default "af")
...
```

File: cmd/fpga_plugin/README.md [hunk 2]
```diff
@@ -57,26 +70,35 @@ $ kubectl describe node <node name> | grep intel.com/fpga
...
+    ```
 
-You may want to modify the file `deployments/fpga_plugin/fpga_plugin.yaml` to
-use your own container image. But the command
-
+3. Build an image from sources:
+    ```
     $ make intel-fpga-plugin
+    ```
+    This image launches `fpga_plugin` in the `af` mode by default.
```
[the remark this answers is not in the document]
Comment (bart0sh) [reply cmd/fpga_plugin/README.md]: yes, 'af' is correct name for this mode.

File: cmd/fpga_plugin/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rojkov) [reply cmd/fpga_plugin/README.md]: Good catch. The correct mode name is `af`.

**Comment by bart0sh** [comment]
@MCamp859 please, resolve conflict and I'll be able to merge it.

**Pull request merged**